### DISCUSSION TITLE
123云盘突然无法挂载

### DISCUSSION THREAD

**Discussion by 5455848** [AlistGo/alist#6317] [Q&A]
123云盘提示：“账号有安全风险，请使用手机号或者微信登录！”
但是即使我另外换了一个其他的账号，依旧相同的提示。

**Comment by AspirantH** [comment]
到官网手动重置一下密码就行了，暂时是这样，但是后续不清楚会不会有别的限制，另一个帖里说要VIP才能挂载webdav了，但我的免费用户目前正常

**Reply by sunny-00001** [reply]
我问过客服，通过alist挂载的123云盘没有限制，通过webdav方式挂载会有限制会员才能用

**Reply by yee1819** [reply]
我试了一下没有用....

**Reply by 5455848** [reply]
好的，我去看看，感谢！

**Reply by realSNES** [reply]
没用

**Comment by ririye** [comment]
打开网页版123网盘，用微信扫码登录下绑定上威信就好了

**Reply by realSNES** [reply]
没用
